utils/python/CIME/SystemTests/rep.py

```diff
@@ -0,0 +1,22 @@
+"""
+Implementation of the CIME REP test
+
+This test verifies that two identical runs give bit-for-bit results
+"""
+
+from CIME.SystemTests.system_tests_compare_two import SystemTestsCompareTwo
+
+class REP(SystemTestsCompareTwo):
```

Comment:
Based on the parsers I've written, I don't think there's any limit on test name length. I'm fine with REP though.